db_stress_tool/db_stress_test_base.cc

```diff
@@ -1264,6 +1274,16 @@ Status StressTest::TestIterate(ThreadState* thread,
   Slice read_ts_slice;
   MaybeUseOlderTimestampForRangeScan(thread, read_ts_str, read_ts_slice, ro);
 
+  std::string op_logs;
+  ro.pin_data = thread->rand.OneIn(2);
+  if (ro.pin_data) {
+    op_logs += "ReadOptions.pin_data = true ";
```

Comment:
Not strictly related to your change, but since you are modifying this area...

I see that op_logs for `total_order_seek`, `auto_prefix_mode`, `iterate_upper_bound`, `iterate_lower_bound` are added _per key_. down in line 1366.

Do we really need them per key? Otherwise, Could we move them up here?

Reply:
Fixed